srv/modules/runners/orderednodes.py

```diff
@@ -15,6 +15,16 @@ def _preserve_order_sorted(seq):
     seen = set()
     return [x for x in seq if x not in seen and not seen.add(x)]
 
+def only(cluster='ceph', role=None):
```

Comment:
Is the purpose of this function to get a list of all minions in the cluster?
If so we already have the select runner. A list of minions can be aquired using the ```select.minions``` runner call.